addons/frag/functions/fnc_dev_drawTrace.sqf

```diff
@@ -0,0 +1,44 @@
+#include "..\script_component.hpp"
+/*
+ * Author: Lambda.Tiger
+ * Per frame function to draw all dev traces.
```

Comment:
```suggestion
 * Per-frame function to draw all dev traces.
```

Reply:
Either version is correct, I only see the hyphenated version in the wiki docs, it's `per frame` every other place in ACE

Reply:
> Either version is correct

I'll wanna see proof of that cuz AFAIK this is the wrong grammar, which is an extremely common problem.

Reply:
So, I use Merrian-Webster) which says [either treatment of compound words is ok/common](https://www.merriam-webster.com/grammar/hyphen-rules-open-closed-compound-words). I also use Grammarly, which backs up not needing it, but on the other hand [Cambridge's grammar](https://dictionary.cambridge.org/grammar/british-grammar/hyphens) guide states supports using the hyphen, albiet not directly.

Either way it comes down to prevelance in ACE is the real decider here:
- [Per-frame](https://github.com/search?q=repo%3Aacemod%2FACE3%20%22per-frame%22&type=code)
- [Per frame](https://github.com/search?q=repo%3Aacemod%2FACE3+%22per+frame%22&type=code)

Reply:
Prevalence doesn't mean it's correct (just like overuse of lazy eval in SQF), like I said. At least M-W is giving actual instructions, but we'll see when I can memorize those ...

Reply:
Looking around, I think part of it is also dialects and maybe how stringent they are on rules versus evolution of the use of language. Cambridge, nominally based in Cambridge, England, would be british english, and claims it's best for intermidiate learners of english / [CFER B1-B2](https://en.wikipedia.org/wiki/Common_European_Framework_of_Reference_for_Languages#Language_activities). Purdue's Online Writing Lab (OWL) also [suggests using a hyphenated compound word](https://owl.purdue.edu/owl/general_writing/punctuation/hyphen_use.html)). Purdue being based in the upper midwest of the U.S.A. and their OWL appears to be targeted at college students. Finally, Merriam-Webster is a dictionary company that publishes and is headquartered in Springfield, Massachusetts, U.S.A (northeast). It is a long-term commercial enterprise focused on dictionaries, including a “Dictionary of English Usage.”

I'll just rewrite this so it isn't a question of correct or incorrect grammar. 
`This function draws all development traces and is intended to be called on each frame.`